pages/initiatives/gsoc/gsoc2023ideas.md

```diff
@@ -121,7 +121,7 @@ You can chose to work on one or a combination of ideas depending on the difficul
 ##### Getting Started
 
 * [BLT Core](https://github.com/owasp/blt/)
-* [BugHeist Mobile App](https://github.com/Bugheist/Flutter/)
+* [Flutter App](https://github.com/Bugheist/Flutter/)
```

Comment:
The Flutter app looks to me like its just for Bugheist.
I dont think thats a suitable GSoC project.

Reply:
Flutter app has been in the GSoC ideas for BLT for the past 3 years with mentees contributing to it, additionally currently there's no commercial usage in BugHeist. Hence I think, it's a suitable one for GSoC. We also intend to remove BugHeist references in the app.

Reply:
Yes we’re going to be correcting the app name to be BLT-Flutter and make it clear to contributors to use the BLT name.  Let’s do that first @just-ary27 and then we can add it back to the PR.  For now let’s take it out.

Reply:
Cool, that sounds good!

Reply:
@psiinon Added the change, mergeable now.